web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -133,7 +141,7 @@ export const SidebarCardsList = (props: Props): ReactElement => {
           {showEmptyState() && (
             <div className="fdc fac margin-y-3">
               <h3 className="text-normal">{Config.dashboardDefaults.emptyOpportunitiesHeader}</h3>
-              <img src={`/img/signpost.svg`} className="" alt="" />
+              <img src={`/img/for-you-section.svg`} className="" alt="" />
```

Comment:
[pebble] you should add the alt text here as well.

Reply:
We specifically don't want alt text here.